performance/src/efficiency/seeding_performance_writer.cpp

```diff
@@ -143,34 +147,75 @@ void seeding_performance_writer::write(
         assert(seed_measurements.size() > 0u);
         if (static_cast<double>(particle_hit_counts.at(0).hit_counts) /
                 static_cast<double>(seed_measurements.size()) >
-            m_cfg.matching_ratio) {
+            m_cfg.truth_config.matching_ratio) {
             auto pid = particle_hit_counts.at(0).ptc.particle_id;
             match_counter[pid]++;
+        } else {
+            total_fake_seeds++;
         }
     }
 
+    std::size_t total_truth_particles = 0;
+    std::size_t total_matched_truth_particles = 0;
+    std::size_t total_duplicate_seeds = 0;
+
     for (auto const& [pid, ptc] : evt_data.m_particle_map) {
+        std::size_t num_measurements = 0;
+
+        // Find the number of measurements belonging to this track
+        if (auto it = evt_data.m_ptc_to_meas_map.find(ptc);
+            it != evt_data.m_ptc_to_meas_map.end()) {
+            num_measurements = it->second.size();
+        } else {
+            continue;
+        }
```

Comment:
Probably more a matter of taste, so please ignore if you want
```suggestion
        // Find the number of measurements belonging to this track
        if (!evt_data.m_ptc_to_meas_map.contains(ptc)) {
             continue;
        }
        num_measurements = it->second.size();
```